### DISCUSSION TITLE
Changing next/image to next/legacy/image and next/future/image to next/image

### DISCUSSION THREAD

**Discussion by Anishpras** [vercel/next.js#40466] [Ideas]
### Describe the feature you'd like to request

As mentioned in the blog - https://nextjs.org/blog/next-12-3. Rename next/image to next/legacy/image and next/future/image to next/image. 

### Describe the solution you'd like

I would like to work on this feature.

### Describe alternatives you've considered

NA

**Comment by styfle** [comment]
I was planning to work on it this week, but I can hold off if you think you can get a PR with tests submitted by the end of the week. Does that sound good?

**Reply by Anishpras** [reply]
@styfle Yes, sure. I would like to work on it.

**Reply by exil0867** [reply]
> I was planning to work on it this week, but I can hold off if you think you can get a PR with tests submitted by the end of the week. Does that sound good?

@styfle Hello, do you have an idea when will the change take effect (approximately)?

**Reply by Anishpras** [reply]
@styfle I am raising the PR with the changes but need help for the tests.

**Reply by Anishpras** [reply]
@styfle I have raised the PR - https://github.com/vercel/next.js/pull/40502

**Reply by styfle** [reply]
@Anishpras I misunderstood, I thought you meant the codemod that renames but you actually mean to rename the source code. In this case we'll have to wait until Next.js 13. Also the tests need to be updated.

**Reply by Anishpras** [reply]
Okay!
